src/token_contract/src/test/mint_to_private.nr

```diff
@@ -20,6 +20,22 @@ unconstrained fn mint_to_private_success() {
     assert(total_supply == mint_amount);
 }
 
+#[test]
+unconstrained fn mint_to_private_stores_commitment() {
+    let (env, token_contract_address, owner, _, minter) =
+        utils::setup_with_minter(/* with_account_contracts */ false);
+    let mint_amount: u128 = 10_000;
+    env.impersonate(minter);
+    Token::at(token_contract_address).mint_to_private(owner, owner, mint_amount).call(
+        &mut env.private(),
+    );
+    // TODO: how do we retrieve the used commitment from here?
+    // assert(
+    //     utils::_commitment_storage_read(token_contract_address, commitment.commitment()) == true,
+    //     "Commitment not stored",
+    // );
+}
```

Comment:
there are 3 minting methods:
- mint to private: in private -> creates a Note and queues total supply increase
- mint to public: in public (can be enqueued from private) -> increases balance & supply
- mint to commitment: in public (can be enqueued) -> completes a partial note & supply++

which one are we testing here?
we should add a test for `mint_to_commitment`